### PR TITLE
COMPANIES CONFERENCE TOOL 

### PR DESCRIPTION
**Pull request by GhislainMitahi**
**COMPANIES CONFERENCE TOOL  Project**

I Built only these 2 pages:

- The home page.

- The about page.

Each of these pages has versions for 2 different screen sizes:

- Mobile: up to 768px wide.

- Desktop: 768px or wider.

Interactions

Links;

- The home page has a link in the menu to the about page.

- The logo in the header links to the home page.

Mobile menu

When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
The mobile menu has a close (X) button that closes the menu.

Dynamic page

The section "Featured speakers" are created dynamically in JavaScript.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,5 +49,5 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is inspired to **Cindy Shin in Behance** template CAMPANY STUFF_Conferance
+This project is inspired to **Cindy Shin in Behance** template CC-Global-Summit
```
Comment (roshan-bajgain) [inline README.md]: - [x] The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly implement it

File: app.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  text: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's",
+};
+
+const speaker4 = {
+  imageSrc: './assets/barack-obama.jpeg',
+  name: 'TRESOR TRESOR',
+  rank: 'GRENERAL DIRECTOR',
+  id: 'fourthSPeaker',
+  text: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's",
+};
```
Comment (roshan-bajgain) [inline app.js:59]: - [x]  It contains a bottom section with a column that includes a section header, a divider, and six individual articles, each with an image, title, subtitle, and description. Each article has two columns, the first with the image at the left and the second with the title, subtitle, and description. you have only 4 kindly go through the requirement
### Your design
his design
### The design provided by microverse
figma

File: about.html [hunk 3]
```diff
@@ -74,21 +116,21 @@ <h2 class="part_title">Partner</h2>
...
         </p>
         </section>
         <section class="second_f_section">
-            Lorem ipsum dolor sit, amet consectetur
+            <h6>Lorem ipsum dolor sit, amet consectetur
             consecteturconsectetur<br>
-            Lorem ipsum dolor sit, amet consectetur
+            Lorem ipsum dolor sit, amet consectetur</h6>
         </section>
     </footer>
```
Comment (roshan-bajgain) [inline about.html]: - [x] The footer has the background color of totally black and it has a logo on the left and text on the right which cover the whole width. Kindly see the design below and make the design similar to it
### Your footer
yourff
### Original design
footer

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @GhislainMitahi, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Suggested changes

- [x] Your mobile version is well but after the more buttons the functionality doesn't work, You need to show the logo and footer in the mobile version as well on the home page as well. Kindly implement it. 
- [x] If you want then you can implement the functionality of more buttons ie: you can make when users click the buttons then they can see more details of the speaker feature.
### your design
mbl
### Logo and footer design that is missing
sssss

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 4]
```diff
@@ -38,15 +41,22 @@ To get a local copy up and running follow these simple example steps.
...
+This project is inspired to: [ Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) template CC-Global-Summit   Cindy Shin, I'm so gratfull for all.
+
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is inspired to **Cindy Shin in Behance** template CAMPANY STUFF_Conferance
+This project is not licensed.
```
Comment (rustam-tolipov) [inline README.md:59]: [OPTIONAL] Why your project is not licensed? I think it would be better to have the MIT license in it.

File: README.md [hunk 5]
```diff
@@ -10,6 +10,8 @@ This project is about to give me a handle as a student on how to create a web si
 
 ## Live demo
 
+https://ghislainmitahi.github.io/conference_cap/
```
Comment (rustam-tolipov) [inline README.md:13]: [OPTIONAL] Please, add a screenshot of the project to your readme.

**Review by rustam-tolipov — APPROVED** [review]
## Hello @GhislainMitahi 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (GhislainMitahi) [reply README.md]: Thank you for the feedback, happy to hear about it and I work on it

File: about.html (hunk 3 above)
Comment (GhislainMitahi) [reply about.html]: Okay thank you, hope I will do it better

File: README.md (hunk 5 above)
Comment (GhislainMitahi) [reply README.md:13]: Sounds Good

File: README.md (hunk 4 above)
Comment (GhislainMitahi) [reply README.md:59]: Okay thank you let me work on it then

**Pull request merged**